iceoryx_hoofs/source/posix_wrapper/semaphore.cpp

```diff
@@ -39,15 +37,21 @@ Semaphore& Semaphore::operator=(Semaphore&& rhs) noexcept
     {
         closeHandle();
 
-        CreationPattern_t::operator=(std::move(rhs));
-
         m_name = std::move(rhs.m_name);
+        // NOLINTNEXTLINE(hicpp-move-const-arg) might break silently if object becomes non-trivial
```

Comment:
Are these linter attributes still necessary? If yes, we need to find a solution which makes them obsolete. It's kind of embarrassing to have these in our code base.